### PR TITLE
Add VladimirYakovlev.ElectronMail version 4.12.4

### PR DESCRIPTION
**Pull request by dyaminigo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20471)

### CODE REVIEW

**Comment by Dyaminigo** [comment]
I don't understand what the validation error is. Installing ElectronMail with the / S flag performs a silent installation, no manual input is required. I cannot reproduce a situation where the installer would ask for some action. On my device, everything is successfully installed in a hidden (quiet) mode with the / S flag.
P.S. I am sorry for my bad english.

File: manifests/v/VladimirYakovlev/ElectronMail/4.12.4/VladimirYakovlev.ElectronMail.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: VladimirYakovlev.ElectronMail
+PackageVersion: 4.12.4
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/vladimiry/ElectronMail/releases/download/v4.12.4/electron-mail-4.12.4-windows-x64-nsis-installer.exe
+  InstallerSha256: A11DF51E3871F76373958FF84579119FECA43FB45F8263D79CFD04BDF8FAE710
+  InstallerSwitches:
+    Silent: /S
```
Comment (ItzLevvie) [inline manifests/v/VladimirYakovlev/ElectronMail/4.12.4/VladimirYakovlev.ElectronMail.installer.yaml:12]: These switches are not needed; winget-cli makes nullsoft installers run with `/S` in the code.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see review.

**Comment by ItzLevvie** [comment]
You'll need to contact the application's developer(s) to allow the Visual C++ Redist to install silently, I'm currently stuck here:
image

**Comment by Dyaminigo** [comment]
Hello @ItzLevvie,
Thank you for comments about the manifest compiled by me and for the problem found in unsuccessful installer checks! I will contact the ElectronMail developer and give him information about the installer. In the meantime, I'll close this PR until the ElectronMail developer will not solve the problem with the installer.

**Pull request closed without merging**